aten/src/ATen/native/xnnpack/Factory.cpp

```diff
@@ -27,7 +29,9 @@ Tensor empty_with_tail_padding(
           },
           DispatchKeySet{DispatchKey::CPUTensorId}));
 
-  return tensor.resize_(size, memory_format);
+  return namedinference::propagate_names_if_nonempty(
+      tensor.resize_(size, memory_format),
+      maybe_names);
```

Comment:
Note: Had to add this to prevent a test in test/test_namedtensor.py from failing.  Said test expects tensor names to propagate.